prepare/engines/classification/classification_engines.py

```diff
@@ -0,0 +1,56 @@
+from unitxt import add_to_catalog
+from unitxt.inference import (
+    IbmGenAiInferenceEngine,
+    OpenAiInferenceEngine,
+    WMLInferenceEngine,
+)
+from unitxt.random_utils import get_seed
+
+
+def get_inference_engine(model_name, framework_name):
+    if framework_name == "ibm_wml":
+        return WMLInferenceEngine(
+            model_name=model_name,
+            max_new_tokens=5,
+            random_seed=get_seed(),
```

Comment:
this will not do what I think you think it will do. The see will be fixed to the one that was set when the json was created.

Reply:
it sets the seed to 42, which is fine by me - I just want it to be deterministic

Reply:
so set it manually. The json are being prepared each pr as part of the test. In it ever change it will cause problems